llvm/test/tools/llvm-gsymutil/X86/elf-dwo.yaml

```diff
@@ -15,8 +15,8 @@
 
 ## WARNING-QUIET: Input file: {{.*\.yaml\.tmp}}
 ## WARNING-QUIET: Output file (x86_64): {{.*\.yaml\.tmp\.gsym}}
-## WARNING-QUIET: warning: Unable to retrieve DWO .debug_info section for some object files. (Remove the --quiet flag for full output)
 ## WARNING-QUIET: Pruned 0 functions, ended with 10 total
+## WARNING-QUIET: warning: Unable to retrieve DWO .debug_info section for some object files. (Remove the --quiet flag for full output)
```

Comment:
If this is output is non deterministic, like some stuff goes to STDERR and some stuff goes to STDOUT, you should add "-DAG" to your lines. So this will always work even if things come out in a different order:
```
## WARNING-QUIET-DAG: Pruned 0 functions, ended with 10 total
## WARNING-QUIET-DAG: warning: Unable to retrieve DWO .debug_info section for some object files. (Remove the --quiet flag for full output)
```

Reply:
It's not nondeterministic. Because the message is now an aggregated title, it's just emitted at a (deterministic, but) different point in the output.